docs/spfx/viva/get-started/making-quickview-compatable-darkmode-mobile.md

```diff
@@ -113,5 +183,3 @@ public get data(): IQuickViewData {
 
 Similarly, we can customize our icons, images, and other elements of Quick View to support both light and dark theme.
 
-> [!NOTE]
```

Comment:
We need to keep this note, just update that variable

Reply:
Done